### ISSUE TITLE
contextual version 3.0.0 prints interpolated text during compile

### ISSUE THREAD

**Issue by balhoff** [propensive/contextual#65]
Very minor issue, but I think there may be some debugging statements left in. For example, with my interpolator:

```
scala> import org.phenoscape.sparql.SPARQLInterpolation._
import org.phenoscape.sparql.SPARQLInterpolation._

scala> val s = sparql"hello"
Literals: 'List(hello)'
s: org.phenoscape.sparql.SPARQLInterpolation.QueryText = QueryText(hello)
```

I don't expect to see `Literals: 'List(hello)'`.

**Comment by propensive** [comment]
Looks like I must have left a `println` in there... I'll sort this out and publish `3.0.1` asap.

**Comment by balhoff** [comment]
@propensive, thanks—I see `3.0.1` but `Prefix` seems to have disappeared again in that release. Is that expected?

**Comment by propensive** [comment]
I'll have a look at what I just published... I guess I've made a mistake with the branch I published from, but for now: hold on for `3.0.2`. :/

**Comment by danslapman** [comment]
Any news on 3.0.2?
